source/main.lua

```diff
@@ -1,155 +1,207 @@
-gstrGameVersion = "0.10"
+
+-- ~~~~~~~~~~~~~~~~~~
+-- Mars Lander (2021)
+-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+-- https://github.com/togfoxy/MarsLander
+-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+gstrGameVersion = "0.11"
+
+love.window.setTitle("Mars Lander " .. gstrGameVersion)
 
 io.stdout:setvbuf("no")
 
+
+-- Do debug stuff like display info text etc
+gbolDebug = true
+
+-- Global screen dimensions
+gintScreenWidth = 1024 -- 1920
+gintScreenHeight = 768 -- 1080
+
+
+
+-- ~~~~~~~~~~~
+-- Libraries
+-- ~~~~~~~~~~~
+

```

Comment:
I wonder if we should take this opportunity to make all the require statements consistant.

Some are require 'xx', some are require "xxx", some are require (" ")

Reply:
I suggest require 'xx'. Easy enough to fix in the next PR.

Reply:
I'm happy with '' as well - as long as it is consistant (maybe I'll add that to the code style as well)